frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -59,7 +59,7 @@ Cypress.Commands.add('testA11y', (target: string) => {
 export const checkErrors = () =>
   cy.window().then((win) => {
     if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
+      throw win.windowError;
```

Comment:
We should fix the types and only assign an `Error` object. Right now it's `boolean | string`, which is wrong. We should initialize to `null` instead of `false`.

Reply:
ok, fixed

Reply:
Set `windowError` type to `PromiseRejectionEvent`.

Reply:
resolved, pls see latest code